app/components/application/site-search/group/item.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+import get from 'ember-metal/get';
+import computed from 'ember-computed';
+import service from 'ember-service/inject';
+import { typeOf } from 'ember-utils';
+import { modelType } from 'client/helpers/model-type';
+import { hrefTo } from 'ember-href-to/helpers/href-to';
+import { invokeAction } from 'ember-invoke-action';
+
+export default Ember.Component.extend({
+  router: service('-routing'),
+
+  link: computed('item', {
```

Comment:
Might need to verify this, but I don't think that `item` would ever be recomputed so should switch to a `getter` here.